Phases/Assessments/README.md

```diff
@@ -0,0 +1,131 @@
+# Phase Assessments
+
+This document explains how phase assessments will work. To learn what each phase assessment will cover and how to prepare, see the phase-specific assessment pages:
+
+- [Phase 1](./Phase_1.md)
+- [Phase 2](./Phase_2.md)
+- [Phase 3](./Phase_3.md)
+
+Note that there is no assessment for phases 4 or 5.
+
+**Table of Contents**
+
+<!-- TOC depthFrom:2 depthTo:3 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Schedule & Pacing](#schedule-pacing)
+- [The Process](#the-process)
+	- [Requesting an Early Assessment](#requesting-an-early-assessment)
+- [Preparing for the Assessment](#preparing-for-the-assessment)
+- [Getting Support](#getting-support)
+- [Evaluation / Interview](#evaluation-interview)
+	- [Artifact Evaluation](#artifact-evaluation)
+	- [Technical Interview](#technical-interview)
+- [Membership Review](#membership-review)
+
+<!-- /TOC -->
+
+## Schedule & Pacing
+
+There are clear pacing expectations for every learner who joins the guild. Specifically, each phase is designed to contain curriculum that should be able to be completed in 6 weeks, but *must* be completed within 8 weeks. This pacing requirement *includes* the 1 week that will be necessary to go through the assessment process.
+
+Learners may choose to complete an assessment at any time during the phase, but, there are two rules:
+
+1. An assessment may only be attempted *twice*.
+2. A learner must *complete and pass* the assessment no later than the 8th week after they begin the phase.
+
+## The Process
+
+**Note:** This process is identical *regardless* of which phase assessment is being attempted.
+
+When a learner feels ready to attempt an assessment, a Moderator will manually assign them to a phase assessment. The lead coach for the project will be an SEP, and learners should have few or no meetings to attend during that week.
+
+The Moderator will send Learners the link to their assessment challenge on Monday morning. Learners will then have **2.5 days** to work on their solution and complete as much as they can. Learners _must submit_ their assessment solution/artifact by **1pm on Wednesday**.
+
+Evaluations and interviews start after lunch on Wednesday and continue until they are done, ideally before 6pm on Friday.
+
+After the evaluation and interviews have been completed, the results will be sent to Membership to make a decision and inform the learner whether they passed.
+
+### Requesting an Early Assessment
+
+If you would like to take the assessment _before_ the 8th week of your phase, you can request an early assessment from the Moderator. The Moderator will attempt to respond to these requests as best they can, but there is no guarantee that every request will be immediately met.
```

Comment:
why are we still referencing a `Moderator`? There is no game for which to moderate. Instead i think this should be changed to "when youre ready to interview for the next phase schedule an interview with that phase instructor" maybe via calendly?